test_elasticsearch/_async/test_server/test_clients.py

```diff
@@ -24,8 +24,8 @@
 
 
 @pytest.mark.parametrize("kwargs", [{"body": {"text": "привет"}}, {"text": "привет"}])
-async def test_indices_analyze_unicode(async_client, kwargs):
```

Comment:
I think I might have introduced this warning:
```
test_elasticsearch/_async/test_server/test_clients.py::test_indices_analyze_unicode[kwargs0]
--
  | /code/elasticsearch-py/.nox/test-3-12/lib/python3.12/site-packages/pytest_asyncio/plugin.py:759: DeprecationWarning: The event_loop fixture provided by pytest-asyncio has been redefined in
  | /code/elasticsearch-py/test_elasticsearch/_async/conftest.py:30
  | Replacing the event_loop fixture with a custom implementation is deprecated
  | and will lead to errors in the future.
  | If you want to request an asyncio event loop with a scope other than function
  | scope, use the "scope" argument to the asyncio mark when marking the tests.
  | If you want to return different types of event loops, use the event_loop_policy
  | fixture.
```
https://buildkite.com/elastic/elasticsearch-py-integration-tests/builds/1638#018f729e-d596-42a8-aa94-6ed2036e0573

but I can also see it for a test that I didn't touch: `
test_elasticsearch/test_transport.py::test_warning_header[headers1]`.

Reply:
Have to look more at event loop policies https://docs.python.org/3/library/asyncio-policy.html

Reply:
We get this warning for Python 3.8 and above. For 3.7 there is no warning. And I added override of the even loop fixture to fix an error with 3.7:
https://github.com/elastic/elasticsearch-py/blob/370975ef6d01692e7626b3ae3be3863363946796/test_elasticsearch/_async/conftest.py#L29-L31

@pquentin This might be the trade-off we have to make here. We live with the new warning until we remove support for Python 3.7. What do you think?

Reply:
Decided to stop testing 3.7.